warehouse-management-service/pkg/log/logger.go

```diff
@@ -0,0 +1,42 @@
+package log
```

Comment:
question: Is this dead code? I couldn't find usages of this anywhere.

suggestion: Can we instead use the `logging` standard library instead of rolling our own logger? We could then upgrade this over time to a structured logger like uber/zap

Reply:
The idea was to have a custom structured logger.
But I do see why at this point a custom logger doesn't make sense, considering how it doesn't do much.

I also see a similar anti-pattern of the implementer declaring an interface here.
I plan on refactoring this to have functions `Error()`, `Info()`, etc. as opposed to passing in the log level to `Log()`. Will be doing this in an upcoming PR however.